src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { TabList, TabPanels } from '.';
+import './Tabs.scss';
+
+interface Tab {
+  name: string;
+  title: string;
+  panel: JSX.Element;
+}
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  tabs: Tab[];
+}
+
+export const Tabs: React.FC<Props> = ({ tabs }) => {
+  const [activePanel, setActivePanel] = useState(tabs[0].name);
+
+  return (
+    <div className="Tabs">
```

Comment:
One thing you wanna make sure you do with all the components in the design system is to pass through the className and the rest props so that these things can be further configured by the consumer.

```tsx
import classNames from 'classnames';

export const Tabs: React.FC<Props> = ({ tabs, className, ...rest }) => {
  // ...
  return (
    <div className={classNames("Tabs", className) {...rest}>
  // ...
}

Reply:
So just this top level element here is needed since that's the only thing that's actually exposed.

Reply:
Good catch! 
Done